config.c

```diff
@@ -647,6 +647,9 @@ static bool apply_style_option(struct mako_style *style, const char *name,
 			binding.action = MAKO_BINDING_DISMISS_GROUP;
 		} else if (strcmp(value, "invoke-default-action") == 0) {
 			binding.action = MAKO_BINDING_INVOKE_DEFAULT_ACTION;
+		} else if (has_prefix(value, "invoke-action ")) {
+			binding.action = MAKO_BINDING_INVOKE_ACTION;
+			binding.action_name = strdup(value + strlen("invoke-action "));
```

Comment:
Instead of using a separate action we could also implement `invoke-default-action` via the new `invoke-action` binding.
It should just require moving `DEFAULT_ACTION_KEY` over to `include/config.h`.

Reply:
Yeah, that sounds like a good idea!

Reply:
Hm, I am getting a double free on `finish_binding()`, likely because of the initial `left` binding now containing something else than `NULL`. Will keep debugging this later.

Reply:
addressed